pymc/smc/sample_smc.py

```diff
@@ -56,50 +56,53 @@ def sample_smc(
 
     Parameters
     ----------
-    draws: int
+    draws : int, default 2000
         The number of samples to draw from the posterior (i.e. last stage). And also the number of
         independent chains. Defaults to 2000.
-    kernel: SMC Kernel used. Defaults to pm.smc.IMH (Independent Metropolis Hastings)
-    start: dict, or array of dict
+    kernel : SMC_kernel, optional
```

Comment:
The `SMC_kernel` alias is not working, because there is a mistake in https://github.com/pymc-devs/pymc/blob/main/docs/source/api/smc.rst. It has:

```
(smc_kernels)=
SMC kernels
-----------
```

which shoud instead be 

```
.. _smc_kernels:

SMC kernels
-----------
```